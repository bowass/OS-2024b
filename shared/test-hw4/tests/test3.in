64145
8010
55674
13689
2879
14657
31810
57892
5294
45611
40231
57530
32015
25193
10043
41247
15029
19312
56039
35144
27976
2594
44800
2847
8269
9316
20728
37724
59338
50607
28793
50114
49195
14233
27552
47550
58758
661
27949
37537
56043
8920
58789
39932
9563
56928
56130
61133
25397
32943
43386
62993
28378
45930
59851
62661
17138
44249
45672
63391
10616
10694
3758
37212
33753
24403
48809
19009
58695
46328
10198
53918
3199
5457
22885
17992
7186
27211
55937
23841
56965
18303
33427
65156
25491
28666
16510
38975
23318
10039
58366
38150
10174
52273
23679
47004
17563
39613
30918
41993
26817
64394
13704
22846
15356
214
62358
13924
36056
49183
57907
23582
61832
64714
26500
47106
64808
26999
25821
53647
11797
7647
32634
24980
8379
22420
64162
30864
15871
8262
62705
18674
15814
42560
23219
3609
17415
17121
64459
42339
27081
16560
4160
47945
25493
20587
28893
3772
27309
51324
42344
19719
20489
58853
38708
49768
62422
7977
36130
32325
38559
65393
21062
12704
16820
22243
37585
2840
7436
45643
60771
28237
20282
32950
34967
5488
61974
18175
14534
49414
34715
23467
27816
45197
25041
23941
1644
14697
27617
41326
28467
41382
34701
26154
2873
29172
28929
21043
15718
52810
57702
28297
24634
48650
21926
55694
61493
34580
59927
45608
53355
8419
17933
29125
7139
5586
58392
6857
62071
46342
3460
3875
31571
25531
62658
9932
2711
38610
8427
24984
26892
11250
1065
6872
27794
3521
57395
2657
42684
44121
54235
5467
25409
38063
7264
41487
47560
61594
23236
2157
62904
32955
23010
34854
19862
14590
23392
62988
16170
33603
30424
65111
54817
19330
58670
46851
48410
59383
2798
175
62906
46299
31375
6374
27786
8635
42321
63950
38814
2688
1726
62522
45668
65120
49829
50
31003
58658
31808
24068
53049
50092
54841
57775
63902
25936
19010
62505
5441
60665
18009
62981
4170
4140
28839
24198
65264
4890
64586
9227
17360
20737
4570
46833
2352
15628
46653
8282
52437
45665
40942
48656
51035
32721
40270
47200
64669
7756
6654
3905
58953
16924
44207
10319
14136
4551
49849
22090
44801
56306
30278
65355
57016
4234
18756
53754
47988
53044
15065
29382
26709
7762
31516
37210
63061
50128
34463
43880
50652
30893
47618
23588
19443
63234
62437
34435
30671
3120
59245
42871
34654
32638
28148
56222
16735
12532
27267
40747
23015
23805
10552
47107
48469
37621
20014
2405
24312
59261
55423
54690
3831
61367
42438
39229
36257
26872
56025
25429
54277
54149
31560
29472
9304
5933
32955
53523
53189
37125
35263
26929
14819
24921
26367
10379
49627
9845
34724
55383
14303
20631
56522
62447
8514
34555
28064
48004
30073
49846
28493
47948
42664
20664
56319
52953
43447
21781
30343
36807
10507
35346
5051
22476
58756
42646
53150
19308
15130
8512
7162
31822
8714
56984
18834
18167
59103
22679
20757
64212
12289
12558
53755
33246
63672
19169
62283
54564
24296
34582
47313
43928
51047
22936
34959
15023
49131
13060
53942
54579
55010
59713
65293
39432
12525
46900
34398
47083
48291
62818
15440
8067
11358
37887
33233
39253
4935
43343
42098
45348
27106
56139