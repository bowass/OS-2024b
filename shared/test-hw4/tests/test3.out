Virtual address: 64145 Physical address: 145 Value: 0
Virtual address: 8010 Physical address: 330 Value: 7
Virtual address: 55674 Physical address: 634 Value: 54
Virtual address: 13689 Physical address: 889 Value: 0
Virtual address: 2879 Physical address: 1087 Value: -49
Virtual address: 14657 Physical address: 1345 Value: 0
Virtual address: 31810 Physical address: 1602 Value: 31
Virtual address: 57892 Physical address: 1828 Value: 0
Virtual address: 5294 Physical address: 2222 Value: 5
Virtual address: 45611 Physical address: 2347 Value: -118
Virtual address: 40231 Physical address: 2599 Value: 73
Virtual address: 57530 Physical address: 3002 Value: 56
Virtual address: 32015 Physical address: 3087 Value: 67
Virtual address: 25193 Physical address: 3433 Value: 0
Virtual address: 10043 Physical address: 3643 Value: -50
Virtual address: 41247 Physical address: 3871 Value: 71
Virtual address: 15029 Physical address: 4277 Value: 0
Virtual address: 19312 Physical address: 4464 Value: 0
Virtual address: 56039 Physical address: 4839 Value: -71
Virtual address: 35144 Physical address: 4936 Value: 0
Virtual address: 27976 Physical address: 5192 Value: 0
Virtual address: 2594 Physical address: 5410 Value: 2
Virtual address: 44800 Physical address: 5632 Value: 0
Virtual address: 2847 Physical address: 1055 Value: -57
Virtual address: 8269 Physical address: 5965 Value: 0
Virtual address: 9316 Physical address: 6244 Value: 0
Virtual address: 20728 Physical address: 6648 Value: 0
Virtual address: 37724 Physical address: 6748 Value: 0
Virtual address: 59338 Physical address: 7114 Value: 57
Virtual address: 50607 Physical address: 7343 Value: 107
Virtual address: 28793 Physical address: 7545 Value: 0
Virtual address: 50114 Physical address: 7874 Value: 48
Virtual address: 49195 Physical address: 7979 Value: 10
Virtual address: 14233 Physical address: 8345 Value: 0
Virtual address: 27552 Physical address: 8608 Value: 0
Virtual address: 47550 Physical address: 8894 Value: 46
Virtual address: 58758 Physical address: 9094 Value: 57
Virtual address: 661 Physical address: 9365 Value: 0
Virtual address: 27949 Physical address: 5165 Value: 0
Virtual address: 37537 Physical address: 9633 Value: 0
Virtual address: 56043 Physical address: 4843 Value: -70
Virtual address: 8920 Physical address: 9944 Value: 0
Virtual address: 58789 Physical address: 9125 Value: 0
Virtual address: 39932 Physical address: 10236 Value: 0
Virtual address: 9563 Physical address: 10331 Value: 86
Virtual address: 56928 Physical address: 10592 Value: 0
Virtual address: 56130 Physical address: 10818 Value: 54
Virtual address: 61133 Physical address: 11213 Value: 0
Virtual address: 25397 Physical address: 11317 Value: 0
Virtual address: 32943 Physical address: 11695 Value: 43
Virtual address: 43386 Physical address: 11898 Value: 42
Virtual address: 62993 Physical address: 12049 Value: 0
Virtual address: 28378 Physical address: 12506 Value: 27
Virtual address: 45930 Physical address: 12650 Value: 44
Virtual address: 59851 Physical address: 13003 Value: 114
Virtual address: 62661 Physical address: 13253 Value: 0
Virtual address: 17138 Physical address: 13554 Value: 16
Virtual address: 44249 Physical address: 13785 Value: 0
Virtual address: 45672 Physical address: 2408 Value: 0
Virtual address: 63391 Physical address: 13983 Value: -25
Virtual address: 10616 Physical address: 14200 Value: 0
Virtual address: 10694 Physical address: 14278 Value: 10
Virtual address: 3758 Physical address: 14510 Value: 3
Virtual address: 37212 Physical address: 14684 Value: 0
Virtual address: 33753 Physical address: 15065 Value: 0
Virtual address: 24403 Physical address: 15187 Value: -44
Virtual address: 48809 Physical address: 15529 Value: 0
Virtual address: 19009 Physical address: 15681 Value: 0
Virtual address: 58695 Physical address: 9031 Value: 81
Virtual address: 46328 Physical address: 16120 Value: 0
Virtual address: 10198 Physical address: 3798 Value: 9
Virtual address: 53918 Physical address: 16286 Value: 52
Virtual address: 3199 Physical address: 16511 Value: 31
Virtual address: 5457 Physical address: 16721 Value: 0
Virtual address: 22885 Physical address: 16997 Value: 0
Virtual address: 17992 Physical address: 17224 Value: 0
Virtual address: 7186 Physical address: 17426 Value: 7
Virtual address: 27211 Physical address: 17739 Value: -110
Virtual address: 55937 Physical address: 4737 Value: 0
Virtual address: 23841 Physical address: 17953 Value: 0
Virtual address: 56965 Physical address: 10629 Value: 0
Virtual address: 18303 Physical address: 18303 Value: -33
Virtual address: 33427 Physical address: 18579 Value: -92
Virtual address: 65156 Physical address: 18820 Value: 0
Virtual address: 25491 Physical address: 11411 Value: -28
Virtual address: 28666 Physical address: 19194 Value: 27
Virtual address: 16510 Physical address: 19326 Value: 16
Virtual address: 38975 Physical address: 19519 Value: 15
Virtual address: 23318 Physical address: 19734 Value: 22
Virtual address: 10039 Physical address: 3639 Value: -51
Virtual address: 58366 Physical address: 20222 Value: 56
Virtual address: 38150 Physical address: 20230 Value: 37
Virtual address: 10174 Physical address: 3774 Value: 9
Virtual address: 52273 Physical address: 20529 Value: 0
Virtual address: 23679 Physical address: 20863 Value: 31
Virtual address: 47004 Physical address: 21148 Value: 0
Virtual address: 17563 Physical address: 21403 Value: 38
Virtual address: 39613 Physical address: 21693 Value: 0
Virtual address: 30918 Physical address: 21958 Value: 30
Virtual address: 41993 Physical address: 22025 Value: 0
Virtual address: 26817 Physical address: 22465 Value: 0
Virtual address: 64394 Physical address: 22666 Value: 62
Virtual address: 13704 Physical address: 904 Value: 0
Virtual address: 22846 Physical address: 16958 Value: 22
Virtual address: 15356 Physical address: 23036 Value: 0
Virtual address: 214 Physical address: 23254 Value: 0
Virtual address: 62358 Physical address: 23446 Value: 60
Virtual address: 13924 Physical address: 23652 Value: 0
Virtual address: 36056 Physical address: 24024 Value: 0
Virtual address: 49183 Physical address: 7967 Value: 7
Virtual address: 57907 Physical address: 1843 Value: -116
Virtual address: 23582 Physical address: 20766 Value: 23
Virtual address: 61832 Physical address: 24200 Value: 0
Virtual address: 64714 Physical address: 24522 Value: 63
Virtual address: 26500 Physical address: 24708 Value: 0
Virtual address: 47106 Physical address: 24834 Value: 46
Virtual address: 64808 Physical address: 25128 Value: 0
Virtual address: 26999 Physical address: 25463 Value: 93
Virtual address: 25821 Physical address: 25821 Value: 0
Virtual address: 53647 Physical address: 25999 Value: 99
Virtual address: 11797 Physical address: 26133 Value: 0
Virtual address: 7647 Physical address: 26591 Value: 119
Virtual address: 32634 Physical address: 26746 Value: 31
Virtual address: 24980 Physical address: 27028 Value: 0
Virtual address: 8379 Physical address: 6075 Value: 46
Virtual address: 22420 Physical address: 27284 Value: 0
Virtual address: 64162 Physical address: 162 Value: 62
Virtual address: 30864 Physical address: 21904 Value: 0
Virtual address: 15871 Physical address: 27647 Value: 127
Virtual address: 8262 Physical address: 5958 Value: 8
Virtual address: 62705 Physical address: 13297 Value: 0
Virtual address: 18674 Physical address: 27890 Value: 18
Virtual address: 15814 Physical address: 27590 Value: 15
Virtual address: 42560 Physical address: 27968 Value: 0
Virtual address: 23219 Physical address: 28339 Value: -84
Virtual address: 3609 Physical address: 14361 Value: 0
Virtual address: 17415 Physical address: 21255 Value: 1
Virtual address: 17121 Physical address: 13537 Value: 0
Virtual address: 64459 Physical address: 22731 Value: -14
Virtual address: 42339 Physical address: 28515 Value: 88
Virtual address: 27081 Physical address: 25545 Value: 0
Virtual address: 16560 Physical address: 19376 Value: 0
Virtual address: 4160 Physical address: 28736 Value: 0
Virtual address: 47945 Physical address: 29001 Value: 0
Virtual address: 25493 Physical address: 11413 Value: 0
Virtual address: 20587 Physical address: 6507 Value: 26
Virtual address: 28893 Physical address: 7645 Value: 0
Virtual address: 3772 Physical address: 14524 Value: 0
Virtual address: 27309 Physical address: 17837 Value: 0
Virtual address: 51324 Physical address: 29308 Value: 0
Virtual address: 42344 Physical address: 28520 Value: 0
Virtual address: 19719 Physical address: 29447 Value: 65
Virtual address: 20489 Physical address: 6409 Value: 0
Virtual address: 58853 Physical address: 9189 Value: 0
Virtual address: 38708 Physical address: 29748 Value: 0
Virtual address: 49768 Physical address: 30056 Value: 0
Virtual address: 62422 Physical address: 23510 Value: 60
Virtual address: 7977 Physical address: 297 Value: 0
Virtual address: 36130 Physical address: 30242 Value: 35
Virtual address: 32325 Physical address: 30533 Value: 0
Virtual address: 38559 Physical address: 30879 Value: -89
Virtual address: 65393 Physical address: 31089 Value: 0
Virtual address: 21062 Physical address: 31302 Value: 20
Virtual address: 12704 Physical address: 31648 Value: 0
Virtual address: 16820 Physical address: 31924 Value: 0
Virtual address: 22243 Physical address: 32227 Value: -72
Virtual address: 37585 Physical address: 9681 Value: 0
Virtual address: 2840 Physical address: 1048 Value: 0
Virtual address: 7436 Physical address: 26380 Value: 0
Virtual address: 45643 Physical address: 2379 Value: -110
Virtual address: 60771 Physical address: 32355 Value: 88
Virtual address: 28237 Physical address: 12365 Value: 0
Virtual address: 20282 Physical address: 32570 Value: 19
Virtual address: 32950 Physical address: 11702 Value: 32
Virtual address: 34967 Physical address: 32919 Value: 37
Virtual address: 5488 Physical address: 16752 Value: 0
Virtual address: 61974 Physical address: 33046 Value: 60
Virtual address: 18175 Physical address: 17407 Value: -65
Virtual address: 14534 Physical address: 33478 Value: 14
Virtual address: 49414 Physical address: 33542 Value: 48
Virtual address: 34715 Physical address: 33947 Value: -26
Virtual address: 23467 Physical address: 19883 Value: -22
Virtual address: 27816 Physical address: 34216 Value: 0
Virtual address: 45197 Physical address: 34445 Value: 0
Virtual address: 25041 Physical address: 27089 Value: 0
Virtual address: 23941 Physical address: 18053 Value: 0
Virtual address: 1644 Physical address: 34668 Value: 0
Virtual address: 14697 Physical address: 1385 Value: 0
Virtual address: 27617 Physical address: 8673 Value: 0
Virtual address: 41326 Physical address: 3950 Value: 40
Virtual address: 28467 Physical address: 18995 Value: -52
Virtual address: 41382 Physical address: 4006 Value: 40
Virtual address: 34701 Physical address: 33933 Value: 0
Virtual address: 26154 Physical address: 34858 Value: 25
Virtual address: 2873 Physical address: 1081 Value: 0
Virtual address: 29172 Physical address: 35316 Value: 0
Virtual address: 28929 Physical address: 35073 Value: 0
Virtual address: 21043 Physical address: 31283 Value: -116
Virtual address: 15718 Physical address: 27494 Value: 15
Virtual address: 52810 Physical address: 35402 Value: 51
Virtual address: 57702 Physical address: 35686 Value: 56
Virtual address: 28297 Physical address: 12425 Value: 0
Virtual address: 24634 Physical address: 35898 Value: 24
Virtual address: 48650 Physical address: 15370 Value: 47
Virtual address: 21926 Physical address: 36262 Value: 21
Virtual address: 55694 Physical address: 654 Value: 54
Virtual address: 61493 Physical address: 36405 Value: 0
Virtual address: 34580 Physical address: 33812 Value: 0
Virtual address: 59927 Physical address: 36631 Value: -123
Virtual address: 45608 Physical address: 2344 Value: 0
Virtual address: 53355 Physical address: 36971 Value: 26
Virtual address: 8419 Physical address: 6115 Value: 56
Virtual address: 17933 Physical address: 17165 Value: 0
Virtual address: 29125 Physical address: 35269 Value: 0
Virtual address: 7139 Physical address: 37347 Value: -8
Virtual address: 5586 Physical address: 16850 Value: 5
Virtual address: 58392 Physical address: 37400 Value: 0
Virtual address: 6857 Physical address: 37833 Value: 0
Virtual address: 62071 Physical address: 33143 Value: -99
Virtual address: 46342 Physical address: 37894 Value: 45
Virtual address: 3460 Physical address: 38276 Value: 0
Virtual address: 3875 Physical address: 38435 Value: -56
Virtual address: 31571 Physical address: 38739 Value: -44
Virtual address: 25531 Physical address: 11451 Value: -18
Virtual address: 62658 Physical address: 13250 Value: 61
Virtual address: 9932 Physical address: 39116 Value: 0
Virtual address: 2711 Physical address: 5527 Value: -91
Virtual address: 38610 Physical address: 30930 Value: 37
Virtual address: 8427 Physical address: 6123 Value: 58
Virtual address: 24984 Physical address: 27032 Value: 0
Virtual address: 26892 Physical address: 25356 Value: 0
Virtual address: 11250 Physical address: 39410 Value: 10
Virtual address: 1065 Physical address: 39465 Value: 0
Virtual address: 6872 Physical address: 37848 Value: 0
Virtual address: 27794 Physical address: 34194 Value: 27
Virtual address: 3521 Physical address: 38337 Value: 0
Virtual address: 57395 Physical address: 2867 Value: 12
Virtual address: 2657 Physical address: 5473 Value: 0
Virtual address: 42684 Physical address: 28092 Value: 0
Virtual address: 44121 Physical address: 13657 Value: 0
Virtual address: 54235 Physical address: 39899 Value: -10
Virtual address: 5467 Physical address: 16731 Value: 86
Virtual address: 25409 Physical address: 11329 Value: 0
Virtual address: 38063 Physical address: 40111 Value: 43
Virtual address: 7264 Physical address: 17504 Value: 0
Virtual address: 41487 Physical address: 40207 Value: -125
Virtual address: 47560 Physical address: 8904 Value: 0
Virtual address: 61594 Physical address: 36506 Value: 60
Virtual address: 23236 Physical address: 28356 Value: 0
Virtual address: 2157 Physical address: 40557 Value: 0
Virtual address: 62904 Physical address: 40888 Value: 0
Virtual address: 32955 Physical address: 11707 Value: 46
Virtual address: 23010 Physical address: 17122 Value: 22
Virtual address: 34854 Physical address: 32806 Value: 34
Virtual address: 19862 Physical address: 29590 Value: 19
Virtual address: 14590 Physical address: 33534 Value: 14
Virtual address: 23392 Physical address: 19808 Value: 0
Virtual address: 62988 Physical address: 12044 Value: 0
Virtual address: 16170 Physical address: 41002 Value: 15
Virtual address: 33603 Physical address: 14915 Value: -48
Virtual address: 30424 Physical address: 41432 Value: 0
Virtual address: 65111 Physical address: 18775 Value: -107
Virtual address: 54817 Physical address: 41505 Value: 0
Virtual address: 19330 Physical address: 4482 Value: 18
Virtual address: 58670 Physical address: 9006 Value: 57
Virtual address: 46851 Physical address: 20995 Value: -64
Virtual address: 48410 Physical address: 41754 Value: 47
Virtual address: 59383 Physical address: 7159 Value: -3
Virtual address: 2798 Physical address: 5614 Value: 2
Virtual address: 175 Physical address: 23215 Value: 43
Virtual address: 62906 Physical address: 40890 Value: 61
Virtual address: 46299 Physical address: 16091 Value: 54
Virtual address: 31375 Physical address: 42127 Value: -93
Virtual address: 6374 Physical address: 42470 Value: 6
Virtual address: 27786 Physical address: 34186 Value: 27
Virtual address: 8635 Physical address: 42683 Value: 110
Virtual address: 42321 Physical address: 28497 Value: 0
Virtual address: 63950 Physical address: 42958 Value: 62
Virtual address: 38814 Physical address: 29854 Value: 37
Virtual address: 2688 Physical address: 5504 Value: 0
Virtual address: 1726 Physical address: 34750 Value: 1
Virtual address: 62522 Physical address: 13114 Value: 61
Virtual address: 45668 Physical address: 2404 Value: 0
Virtual address: 65120 Physical address: 18784 Value: 0
Virtual address: 49829 Physical address: 30117 Value: 0
Virtual address: 50 Physical address: 23090 Value: 0
Virtual address: 31003 Physical address: 43035 Value: 70
Virtual address: 58658 Physical address: 8994 Value: 57
Virtual address: 31808 Physical address: 1600 Value: 0
Virtual address: 24068 Physical address: 43268 Value: 0
Virtual address: 53049 Physical address: 43577 Value: 0
Virtual address: 50092 Physical address: 7852 Value: 0
Virtual address: 54841 Physical address: 41529 Value: 0
Virtual address: 57775 Physical address: 35759 Value: 107
Virtual address: 63902 Physical address: 42910 Value: 62
Virtual address: 25936 Physical address: 43856 Value: 0
Virtual address: 19010 Physical address: 15682 Value: 18
Virtual address: 62505 Physical address: 13097 Value: 0
Virtual address: 5441 Physical address: 16705 Value: 0
Virtual address: 60665 Physical address: 44281 Value: 0
Virtual address: 18009 Physical address: 17241 Value: 0
Virtual address: 62981 Physical address: 12037 Value: 0
Virtual address: 4170 Physical address: 28746 Value: 4
Virtual address: 4140 Physical address: 28716 Value: 0
Virtual address: 28839 Physical address: 7591 Value: 41
Virtual address: 24198 Physical address: 43398 Value: 23
Virtual address: 65264 Physical address: 18928 Value: 0
Virtual address: 4890 Physical address: 44314 Value: 4
Virtual address: 64586 Physical address: 24394 Value: 63
Virtual address: 9227 Physical address: 6155 Value: 2
Virtual address: 17360 Physical address: 44752 Value: 0
Virtual address: 20737 Physical address: 44801 Value: 0
Virtual address: 4570 Physical address: 45274 Value: 4
Virtual address: 46833 Physical address: 45553 Value: 0
Virtual address: 2352 Physical address: 45616 Value: 0
Virtual address: 15628 Physical address: 27404 Value: 0
Virtual address: 46653 Physical address: 45373 Value: 0
Virtual address: 8282 Physical address: 5978 Value: 8
Virtual address: 52437 Physical address: 20693 Value: 0
Virtual address: 45665 Physical address: 2401 Value: 0
Virtual address: 40942 Physical address: 46062 Value: 39
Virtual address: 48656 Physical address: 15376 Value: 0
Virtual address: 51035 Physical address: 46171 Value: -42
Virtual address: 32721 Physical address: 26833 Value: 0
Virtual address: 40270 Physical address: 2638 Value: 39
Virtual address: 47200 Physical address: 24928 Value: 0
Virtual address: 64669 Physical address: 24477 Value: 0
Virtual address: 7756 Physical address: 46412 Value: 0
Virtual address: 6654 Physical address: 46846 Value: 6
Virtual address: 3905 Physical address: 38465 Value: 0
Virtual address: 58953 Physical address: 46921 Value: 0
Virtual address: 16924 Physical address: 13340 Value: 0
Virtual address: 44207 Physical address: 13743 Value: 43
Virtual address: 10319 Physical address: 47183 Value: 19
Virtual address: 14136 Physical address: 8248 Value: 0
Virtual address: 4551 Physical address: 45255 Value: 113
Virtual address: 49849 Physical address: 30137 Value: 0
Virtual address: 22090 Physical address: 32074 Value: 21
Virtual address: 44801 Physical address: 5633 Value: 0
Virtual address: 56306 Physical address: 10994 Value: 54
Virtual address: 30278 Physical address: 41286 Value: 29
Virtual address: 65355 Physical address: 31051 Value: -46
Virtual address: 57016 Physical address: 10680 Value: 0
Virtual address: 4234 Physical address: 28810 Value: 4
Virtual address: 18756 Physical address: 47428 Value: 0
Virtual address: 53754 Physical address: 26106 Value: 52
Virtual address: 47988 Physical address: 29044 Value: 0
Virtual address: 53044 Physical address: 43572 Value: 0
Virtual address: 15065 Physical address: 4313 Value: 0
Virtual address: 29382 Physical address: 47814 Value: 28
Virtual address: 26709 Physical address: 22357 Value: 0
Virtual address: 7762 Physical address: 46418 Value: 7
Virtual address: 31516 Physical address: 38684 Value: 0
Virtual address: 37210 Physical address: 14682 Value: 36
Virtual address: 63061 Physical address: 12117 Value: 0
Virtual address: 50128 Physical address: 7888 Value: 0
Virtual address: 34463 Physical address: 48031 Value: -89
Virtual address: 43880 Physical address: 48232 Value: 0
Virtual address: 50652 Physical address: 7388 Value: 0
Virtual address: 30893 Physical address: 21933 Value: 0
Virtual address: 47618 Physical address: 48386 Value: 46
Virtual address: 23588 Physical address: 20772 Value: 0
Virtual address: 19443 Physical address: 4595 Value: -4
Virtual address: 63234 Physical address: 13826 Value: 61
Virtual address: 62437 Physical address: 23525 Value: 0
Virtual address: 34435 Physical address: 48003 Value: -96
Virtual address: 30671 Physical address: 48847 Value: -13
Virtual address: 3120 Physical address: 16432 Value: 0
Virtual address: 59245 Physical address: 7021 Value: 0
Virtual address: 42871 Physical address: 49015 Value: -35
Virtual address: 34654 Physical address: 33886 Value: 33
Virtual address: 32638 Physical address: 26750 Value: 31
Virtual address: 28148 Physical address: 5364 Value: 0
Virtual address: 56222 Physical address: 10910 Value: 54
Virtual address: 16735 Physical address: 31839 Value: 87
Virtual address: 12532 Physical address: 49396 Value: 0
Virtual address: 27267 Physical address: 17795 Value: -96
Virtual address: 40747 Physical address: 45867 Value: -54
Virtual address: 23015 Physical address: 17127 Value: 121
Virtual address: 23805 Physical address: 20989 Value: 0
Virtual address: 10552 Physical address: 14136 Value: 0
Virtual address: 47107 Physical address: 24835 Value: 0
Virtual address: 48469 Physical address: 41813 Value: 0
Virtual address: 37621 Physical address: 9717 Value: 0
Virtual address: 20014 Physical address: 49454 Value: 19
Virtual address: 2405 Physical address: 45669 Value: 0
Virtual address: 24312 Physical address: 43512 Value: 0
Virtual address: 59261 Physical address: 7037 Value: 0
Virtual address: 55423 Physical address: 49791 Value: 31
Virtual address: 54690 Physical address: 50082 Value: 53
Virtual address: 3831 Physical address: 14583 Value: -67
Virtual address: 61367 Physical address: 50359 Value: -19
Virtual address: 42438 Physical address: 28614 Value: 41
Virtual address: 39229 Physical address: 50493 Value: 0
Virtual address: 36257 Physical address: 30369 Value: 0
Virtual address: 26872 Physical address: 22520 Value: 0
Virtual address: 56025 Physical address: 4825 Value: 0
Virtual address: 25429 Physical address: 11349 Value: 0
Virtual address: 54277 Physical address: 50693 Value: 0
Virtual address: 54149 Physical address: 39813 Value: 0
Virtual address: 31560 Physical address: 38728 Value: 0
Virtual address: 29472 Physical address: 50976 Value: 0
Virtual address: 9304 Physical address: 6232 Value: 0
Virtual address: 5933 Physical address: 51245 Value: 0
Virtual address: 32955 Physical address: 11707 Value: 46
Virtual address: 53523 Physical address: 25875 Value: 68
Virtual address: 53189 Physical address: 43717 Value: 0
Virtual address: 37125 Physical address: 14597 Value: 0
Virtual address: 35263 Physical address: 5055 Value: 111
Virtual address: 26929 Physical address: 25393 Value: 0
Virtual address: 14819 Physical address: 1507 Value: 120
Virtual address: 24921 Physical address: 26969 Value: 0
Virtual address: 26367 Physical address: 35071 Value: -65
Virtual address: 10379 Physical address: 47243 Value: 34
Virtual address: 49627 Physical address: 33755 Value: 118
Virtual address: 9845 Physical address: 39029 Value: 0
Virtual address: 34724 Physical address: 33956 Value: 0
Virtual address: 55383 Physical address: 49751 Value: 21
Virtual address: 14303 Physical address: 8415 Value: -9
Virtual address: 20631 Physical address: 6551 Value: 37
Virtual address: 56522 Physical address: 51658 Value: 55
Virtual address: 62447 Physical address: 23535 Value: -5
Virtual address: 8514 Physical address: 42562 Value: 8
Virtual address: 34555 Physical address: 48123 Value: -66
Virtual address: 28064 Physical address: 5280 Value: 0
Virtual address: 48004 Physical address: 29060 Value: 0
Virtual address: 30073 Physical address: 51833 Value: 0
Virtual address: 49846 Physical address: 30134 Value: 48
Virtual address: 28493 Physical address: 19021 Value: 0
Virtual address: 47948 Physical address: 29004 Value: 0
Virtual address: 42664 Physical address: 28072 Value: 0
Virtual address: 20664 Physical address: 6584 Value: 0
Virtual address: 56319 Physical address: 11007 Value: -1
Virtual address: 52953 Physical address: 35545 Value: 0
Virtual address: 43447 Physical address: 11959 Value: 109
Virtual address: 21781 Physical address: 36117 Value: 0
Virtual address: 30343 Physical address: 41351 Value: -95
Virtual address: 36807 Physical address: 52167 Value: -15
Virtual address: 10507 Physical address: 14091 Value: 66
Virtual address: 35346 Physical address: 52242 Value: 34
Virtual address: 5051 Physical address: 44475 Value: -18
Virtual address: 22476 Physical address: 27340 Value: 0
Virtual address: 58756 Physical address: 9092 Value: 0
Virtual address: 42646 Physical address: 28054 Value: 41
Virtual address: 53150 Physical address: 43678 Value: 51
Virtual address: 19308 Physical address: 4460 Value: 0
Virtual address: 15130 Physical address: 22810 Value: 14
Virtual address: 8512 Physical address: 42560 Value: 0
Virtual address: 7162 Physical address: 37370 Value: 6
Virtual address: 31822 Physical address: 1614 Value: 31
Virtual address: 8714 Physical address: 9738 Value: 8
Virtual address: 56984 Physical address: 10648 Value: 0
Virtual address: 18834 Physical address: 47506 Value: 18
Virtual address: 18167 Physical address: 17399 Value: -67
Virtual address: 59103 Physical address: 47071 Value: -73
Virtual address: 22679 Physical address: 52631 Value: 37
Virtual address: 20757 Physical address: 44821 Value: 0
Virtual address: 64212 Physical address: 212 Value: 0
Virtual address: 12289 Physical address: 49153 Value: 0
Virtual address: 12558 Physical address: 31502 Value: 12
Virtual address: 53755 Physical address: 26107 Value: 126
Virtual address: 33246 Physical address: 52958 Value: 32
Virtual address: 63672 Physical address: 53176 Value: 0
Virtual address: 19169 Physical address: 15841 Value: 0
Virtual address: 62283 Physical address: 23371 Value: -46
Virtual address: 54564 Physical address: 49956 Value: 0
Virtual address: 24296 Physical address: 43496 Value: 0
Virtual address: 34582 Physical address: 33814 Value: 33
Virtual address: 47313 Physical address: 25041 Value: 0
Virtual address: 43928 Physical address: 48280 Value: 0
Virtual address: 51047 Physical address: 46183 Value: -39
Virtual address: 22936 Physical address: 17048 Value: 0
Virtual address: 34959 Physical address: 32911 Value: 35
Virtual address: 15023 Physical address: 4271 Value: -85
Virtual address: 49131 Physical address: 53483 Value: -6
Virtual address: 13060 Physical address: 53508 Value: 0
Virtual address: 53942 Physical address: 16310 Value: 52
Virtual address: 54579 Physical address: 49971 Value: 76
Virtual address: 55010 Physical address: 41698 Value: 53
Virtual address: 59713 Physical address: 12865 Value: 0
Virtual address: 65293 Physical address: 30989 Value: 0
Virtual address: 39432 Physical address: 21512 Value: 0
Virtual address: 12525 Physical address: 49389 Value: 0
Virtual address: 46900 Physical address: 21044 Value: 0
Virtual address: 34398 Physical address: 47966 Value: 33
Virtual address: 47083 Physical address: 21227 Value: -6
Virtual address: 48291 Physical address: 53923 Value: 40
Virtual address: 62818 Physical address: 40802 Value: 61
Virtual address: 15440 Physical address: 54096 Value: 0
Virtual address: 8067 Physical address: 387 Value: -32
Virtual address: 11358 Physical address: 54366 Value: 11
Virtual address: 37887 Physical address: 6911 Value: -1
Virtual address: 33233 Physical address: 52945 Value: 0
Virtual address: 39253 Physical address: 50517 Value: 0
Virtual address: 4935 Physical address: 44359 Value: -47
Virtual address: 43343 Physical address: 11855 Value: 83
Virtual address: 42098 Physical address: 22130 Value: 41
Virtual address: 45348 Physical address: 54564 Value: 0
Virtual address: 27106 Physical address: 25570 Value: 26
Virtual address: 56139 Physical address: 10827 Value: -46
Number of Translated Addresses = 500
Page Faults = 214
Page Fault Rate = 0.428
TLB Hits = 28
TLB Hit Rate = 0.056
